### ISSUE TITLE
cer data on  full LibriSpeech dataset GPU&&CPU

### ISSUE THREAD

**Issue by rongzha1** [samsungsds-rnd/deepspeech.mxnet#9]
Hello,  can you provide cer data on  full LibriSpeech dataset ?  Both GPU and CPU are welcome.
Thanks very much

**Comment by Soonhwan-Kwon** [comment]
Although it's data from two month ago, and we will release sequence-wise batchnorm version soon,
it's cer result for test_clean data with full LibriSpeech dataset( of course train without test_clean data).
Validation result:
Epoch[0] unfortunately powered off when 19 * 3000 batches(batch size is 12)
Epoch[1] 0.177740  (we restarted from checkpoint of 19*3000th batch)
Epoch[2] 0.144390 
Epoch[3] 0.126324
Epoch[4] 0.121056
Epoch[5] 0.110635
Epoch[6] 0.102347
Epoch[7] 0.100333
Epoch[8] 0.098945
(test clean dataset and wav file with shorter than 16 second limitation).
Hope it will help and it is not the best cer and it is just the result of what our time let us to.

**Comment by rongzha1** [comment]
thanks very much. Hope to see your new release soon!
Some data in LibriSpeech/train-clean-100  is larger than 16, the max is 24.525. So need to discard data > 16?
Did you test it in CPU?  I try to run it on CPU, CER is 0.7 .

**Comment by Soonhwan-Kwon** [comment]
Yes there is very little portion between 16 and 25seconds but model should be 25/16 bigger so we limited duration.  And we tested it on 3 gpus(gtx1080). I don't think it is possible to run train on cpu because it takes too much time if you train on full LibriSpeech data.
